common/mount.py

```diff
@@ -143,14 +143,8 @@ def __init__(self,
                  profile_id = None,
                  tmp_mount = False,
                  parent = None):
-        self.config = cfg
-        if self.config is None:
-            self.config = config.Config()
-
-        self.profile_id = profile_id
-        if self.profile_id is None:
-            self.profile_id = self.config.currentProfile()
-
+        self.config = cfg if cfg is not None else config.Config()
```

Comment:
Regarding your solution in `common/encfstools.py` you have used `or`. But here you do use a oneliner-if. Is there a reason?

Reply:
just forgot to change it to use `or`. will update